wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php

```diff
@@ -318,7 +317,7 @@ function displayPagenumbers($chosenLetter, $totalEntries, $entriesPerPage, $lang
 		{
 			if($requestname == "semnumber")
 			{
-				$display .= "<li><a href=\"#\" onclick=\"displayEntry('-', '" . $chosenLetter . "', " . ($currentPage - 1) . ");\">" . $prevpage . "</a></li> ";
+				$display .= "<li " . $class . "><a href=\"?semdomain=" . $languagecode . "&semnumber=" . $chosenLetter . "&pagenr=" . ($currentPage - 1) . "\">" . $prevpage . "</a></li> ";
```

Comment:
The variable `$class` is undefined in this function.

Reply:
nice catch! removed, since $class is necessary only for current page